tests/Unit/IO/Observers/Test_ObservationId.cpp

```diff
@@ -81,11 +55,11 @@ SPECTRE_TEST_CASE("Unit.IO.Observers.ObservationId", "[Unit][Observers]") {
   CHECK(id4.observation_type_hash() != id3.observation_type_hash());
   CHECK(id4.value() == 8.0);
 
-  ObservationId time0(Time(Slab{0.0, 1.0}, 0),
+  ObservationId time0(Time(Slab{0.0, 1.0}, 0).value(),
```

Comment:
This section (everything with the `timeN` objects) is a duplicate of the preceding section now.